backend/api/tests/views/test_views_algo.py

```diff
@@ -77,8 +101,14 @@ def setUp(self):
                     "checksum": "dummy-checksum",
                     "storage_address": f"http://testserver/algo/{simple_algo.key}/file/",
                 },
-                "inputs": factory.ALGO_INPUTS_PER_CATEGORY["ALGO_SIMPLE"],
-                "outputs": factory.ALGO_OUTPUTS_PER_CATEGORY["ALGO_SIMPLE"],
+                "inputs": {
+                    "datasamples": {"kind": "ASSET_DATA_SAMPLE", "optional": False, "multiple": True},
+                    "opener": {"kind": "ASSET_DATA_MANAGER", "optional": False, "multiple": False},
+                    "model": {"kind": "ASSET_MODEL", "optional": True, "multiple": False},
+                },
+                "outputs": {
+                    "model": {"kind": "ASSET_MODEL", "multiple": False},
+                },
```

Comment:
Even if it's a bit more verbose, it's better to have an explicit overview of the response we expect IMO. It helps to see the impact of future changes.